CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.13.1 (2020-11-06)
+
+- Fixed missing CSS import. [#194](https://github.com/blackbaud/skyux-theme/pull/194)
```

Comment:
```suggestion
- Added a missing CSS import. [#194](https://github.com/blackbaud/skyux-theme/pull/194)
```

Reply:
@blackbaud-johnly It was actually a removal. I'll make the change, if that works for you?

Reply:
Ok, I tweaked it a bit to better explain. Let me know if this can be improved.